### ISSUE TITLE
Raspberry pi 1 b+ 

### ISSUE THREAD

**Issue by Snowlav** [wizardyhnr/phantomjs-armhf#1]
Hey will your compiled version run on my raspberry pi 1 b+ ? (ARM6 architecture iirc)

**Comment by wizardyhnr** [comment]
No plan for this right now. Raspberry pi community is big enough. I
remember there are some builds for armv6 structure though.

**Comment by Snowlav** [comment]
None of them are suitable though.
Right now I upgraded to a Raspberry PI 3 but the build is failing due to a QT Webkit error.

There is no latest version available and working with the latest version of Raspbian "Jessie".

I don't mind compiling / building myself but like previously explained, the QT Webkit error is preventing a successful build.

**Comment by wizardyhnr** [comment]
that is strange. I believe phantomjs build script has included QT sources
into building process since version 2.1.1. they have got a lot of complains
before due to QT versions on different system are quite difficult to
control. So actually the build script would build necessary QT environment
first.

Did you compile it on your Raspberry Pi and follow the instructions show at
http://phantomjs.org/build.html?

2016-03-10 4:28 GMT-05:00 Snowlav notifications@github.com:

**Comment by Snowlav** [comment]
Correct, this should be the case. And yes I followed the guide on their webpage.

**Comment by wizardyhnr** [comment]
It is hard to say what causes this. But before there are a lot of successful compiling cases on Raspberry Pi 2 with Phantomjs 1.9.8 or even 2.0.

So I guess maybe you can try to compile 2.0 on you SBC following the instructions from http://raspberrypimaker.com/how-to-compile-phantomjs-on-the-raspberry-pi-2/. If it works, then it suggests 2.1.1 has some issues to support "Jessie". If it still fails, then "Jessie" could be the one to be blamed. Then switching to standard ubuntu 14.04 could be an option.

The corresponding src packages can be download from
https://github.com/ariya/phantomjs/releases.

I am using Odroid u3 instead of Paspberry Pi. So I don't have the environment to compile the binary for  Paspbian.

**Comment by Snowlav** [comment]
I've rocked 2.0.1 for a bit but the fundamental problem I was running into was that the website I was trying to load, did not, the package consists of dynamically generated HTML by the use of Javascript. (yes the ENTIRE site).

Whatever I tried it just kept showing up as a blank (transparent) page on the screenshots, even if I waited 5+ minutes (trying to " wait " till it was done loading).

I tried disabling the SSL options and everything that has been posted or discussed on github and stackoverflow.

However on my main rig it runs fine with the latest stable version, which made me think it might be some sort of socket issue since I remember reading that that was an issue and not later added untill either the current stable version or the one before that.

I might be spewing complete nonsense, but that's as far as I've come so far, and my reason for wanting the latest stable version of phantomJS, however I'd be more than fine using 2.0.1 if the transparency / not rendering issue is due to something else.
